Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -603,6 +617,16 @@ namespace AzToolsFramework
             return false;
         }
 
+        bool PrefabPublicHandler::IsOverrideEditing(const Instance& owningInstance) const
+        {
+            AzFramework::EntityContextId editorEntityContextId = AzFramework::EntityContextId::CreateNull();
+            EditorEntityContextRequestBus::BroadcastResult(editorEntityContextId, &EditorEntityContextRequests::GetEditorEntityContextId);
+            InstanceOptionalReference focusedInstance = m_prefabFocusHandler.GetFocusedPrefabInstance(editorEntityContextId);
+            AZ_Assert(focusedInstance.has_value(), "PrefabPublicHandler::IsOverrideEditing - The focused instance is null.");
+
+            return &(focusedInstance->get()) != &owningInstance;
```

Comment:
If the prefab you passed is not the owning instance, it doesn't mean that it'll be an override if you modify it. The instance can be outside of the focus hierarchy too. I think if you rename this function to IsFocusedInstance(), it'll be more accurate to what you are doing here.

Reply:
I think I can reuse this function without creating a new API.

```cpp
bool PrefabFocusHandler::IsOwningPrefabBeingFocused(AZ::EntityId entityId) const;
```